### PR TITLE
Fixed issue GH-30 by ensuring that any imported notes are always processed regardless of if calendars are being imported.

### LINKED ISSUE #30: [BUG] Importing notes only does not work
Hello. Thank you again for all the work bringing this into V13 effectively. 

I have tried using the export/import feature for notes only but it does not seem to be working. Just seeing if I am doing something wrong or if there is another way to bring those notes in. 

I have 5 years of notes chronicling the history of this one campaign and manually readding them is a monumental task so the import option is optimal.

### PR DESCRIPTION
**Pull request by Arctis-Fireblight**
Both SC and SCR support exporting out only the notes for calendars without the calendar itself.
However, the the import logic for SC was that so notes were only processed while importing calendars, thus creating a situation where backing up and restoring only notes just flat out does not work.

This PR separates out the logic for importing calendars and notes, so that they are can now truly be exported and imported independently of each other when importing into SCR.

Fixes #30 

## Summary by CodeRabbit

* **Bug Fixes**
  * Resolved an issue preventing note imports when no calendars were selected.
  * Notes can now be imported independently from calendar imports.
  * Improved destination calendar matching during import to better handle edge cases.

* **Tests**
  * Updated tests to align with the revised import behavior.

### CODE REVIEW

File: src/classes/applications/configuration-app.ts [hunk 1]
```diff
@@ -1834,54 +1832,64 @@ export default class ConfigurationApp extends FormApplication {
...
-                                    journalEntryData.flags["core"].sheetClass = ModuleName + ".NoteSheet";
-                                    journalEntryData.flags[LegacyModuleName] = undefined;
-                                }
+                    )
+                        continue;
+                    const importInto = getTextInputValue(
+                        `.fsc-import-export .fsc-importing .fsc-file-details select[data-for-cal="${calId}-notes"]`,
+                        calId,
+                        this.appWindow
+                    );
```
[the remark this answers is not in the document]
Comment (Arctis-Fireblight) [reply src/classes/applications/configuration-app.ts:1857]: Should be fixed now, good catch. I dropped the note suffix.

File: src/classes/applications/configuration-app.ts [hunk 2]
```diff
@@ -1834,54 +1832,57 @@ export default class ConfigurationApp extends FormApplication {
...
+                }
+            }
+            // Always check for imported notes regardless of if we are importing calendars. Fix Issue GH-30
+            if (Object.prototype.hasOwnProperty.call(data, "notes")) {
+                await NManager.createJournalDirectory();
+                for (const noteName of Object.keys(data.notes)) {
+                    const calId = noteName;
+                    // Do not import the notes if the user did not check the box
+                    if (!getCheckBoxInputValue(`.fsc-import-export .fsc-importing .fsc-file-details input[data-id="${calId}"]`, true, this.appWindow))
+                        continue;
```
[the remark this answers is not in the document]
Comment (Arctis-Fireblight) [reply src/classes/applications/configuration-app.ts:1846]: This was a mistake on my part, I dropped the notes suffix from the wrong line while trying to fix another problem. I have reverted this.
